tests/test_cli.py

```diff
@@ -1,17 +1,136 @@
-from hrqb.cli import main
+from _pytest.logging import LogCaptureFixture
+from click.testing import Result
+
+from hrqb import cli
 
 OKAY_RESULT_CODE = 0
 MISSING_CLICK_ARG_RESULT_CODE = 2
 
 
+def text_in_logs_or_stdout(text, caplog: LogCaptureFixture, result: Result):
```

Comment:
Clever method!

Reply:
I did _not_ want to untangle why `pytest` was hot-swapping `result.output` and `caplog.text` when ultimately it wasn't much of an issue.  I think this method will also serve as a reminder if we do find that logs are scrambled at all in Cloudwatch, that here could lie the issue.